Dockerfile.Nextcloud

```diff
@@ -3,7 +3,9 @@ ARG INSTALL_XDEBUG=1
 
 FROM nextcloud:${NEXTCLOUD_VERSION}
 
-RUN apt update && apt install -y \
+SHELL [ "/bin/bash", "-c" ]
```

Comment:
this was the reason why the if condition while building was not working. the docker build commands do run in sh